vmwc/__init__.py

```diff
@@ -663,6 +663,27 @@ def send_file(self, source_path, guest_path):
         if not resp.status_code == 200:
             raise Exception("Error while uploading file")
 
+    def download_file(self, guest_path, local_path):
+        if self._tools_credentials is None:
+            raise Exception("Login required to use guest functions")
+
+        fti = self._client.get_guest_operations_manager().fileManager.InitiateFileTransferFromGuest(vm=self._raw_virtual_machine,
+                                                                                                    auth=self._tools_credentials,
+                                                                                                    guestFilePath=guest_path)
+
+        resp = requests.get(fti.url.replace("https://*", "https://" + self._client.host), verify=False)
+
+        if not resp.status_code == 200:
+            raise Exception("Error while downloading file")
+
+        size = 0
+        with open(local_path, 'wb') as f:
+            for chunk in resp.iter_content(chunk_size=1024):
```

Comment:
the `1024` can be placed as an optional parameter so the user may change it when required;

```
def download_file(self, guest_path, local_path, chunk_size=1024)
```